From 9d684c0b2f0068274cc7eb85222f45dba5b561b4 Mon Sep 17 00:00:00 2001
From: Sergei Shtepa <sergei.shtepa@veeam.com>
Date: Fri, 9 Dec 2022 15:09:25 +0100
Subject: [PATCH v2 03/21] documentation, capability: fix Generic Block Device
 Capability

When adding documentation for blkfilter, new lines of documentation
appeared in the file include/linux/blkdev.h. To preserve the appearance
of this document, the required sections and function descriptions were
explicitly specified.

Signed-off-by: Sergei Shtepa <sergei.shtepa@veeam.com>
---
 Documentation/block/capability.rst | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/block/capability.rst b/Documentation/block/capability.rst
index 2ae7f064736a..8fad791980bb 100644
--- a/Documentation/block/capability.rst
+++ b/Documentation/block/capability.rst
@@ -8,3 +8,6 @@ This file documents the sysfs file ``block/<disk>/capability``.
 capabilities a specific block device supports:
 
 .. kernel-doc:: include/linux/blkdev.h
+	:DOC: genhd capability flags
+.. kernel-doc:: include/linux/blkdev.h
+	:functions: disk_openers blk_alloc_disk bio_end_io_acct
-- 
2.20.1

